snapshot-9.0.1.yaml

```diff
@@ -0,0 +1,18 @@
+resolver: nightly-2021-01-27
+compiler: ghc-9.0.0.20201227
+
+packages:
+- git: https://github.com/tweag/distributed-closure
+  commit: b92e75ec81e646703c7bde4f578a7352ee34f1ad
+- git: http://github.com/ekmett/exceptions.git
+  commit: d7b742dc129790778f7b6d3347af80c8d69f8fcd
+- git: http://github.com/haskell/vector.git
+  commit: d919ffc72043af0066674aeaa997919e22d2a00f
+- git: http://github.com/facundominguez/async.git
+  commit: c7c61ba7c6af4bde27eed21d44b3770264bca824
+- git: http://github.com/facundominguez/hashable.git
+  commit: a03e06b79e5de3ad6d39522e08d1db830dc2a145
+- git: http://github.com/haskell/primitive.git
+  commit: 87b50ced840b3b1adbd61fe709b2e042f06e6ff3
+- git: http://github.com/tweag/linear-base.git
+  commit: 5af88c64aa5d8ac338098c033ee21616fce087ff
```

Comment:
Stack supports a specific [`github` package type](https://docs.haskellstack.org/en/stable/pantry/#git-and-mercurial-repos). The advantage over `git` is that it downloads the tarball over http rather than the repo through git and is therefore eligible to Bazel's repository caching if you use `stack_snapshot` pinning. E.g.
```
- github: tweag/distributed-closure
  commit: b92e75ec81e646703c7bde4f578a7352ee34f1ad
```

Reply:
Done.